web/templates/recent_topic_row.hbs

```diff
@@ -26,7 +26,7 @@
             {{/if}}
         </div>
     </td>
-    <td class="recent_topic_name">
+    <td class="recent_topic_name" colspan="2">
```

Comment:
Do you mind shedding some light on the reason behind this change? Just trying to get a better understanding.

Reply:
This is the core change of this PR. We needed a way to preserve how the table looks right now (not make a separate column for unread counts) while adding the new `sort by unread` functionality. This was the best way to do this that I can think of.